source/macos/AdaptiveCards/AdaptiveCardsTests/Views/ACRViewTests.swift

```diff
@@ -357,6 +357,11 @@ private class FakeInputHandlingView: NSView, InputHandlingViewProtocol {
     var key: String = NSUUID().uuidString
     var isValid: Bool = true
     var errorMessageHandler: ErrorMessageHandlerDelegate?
+    var isRequired = false
+    var expectation: XCTestExpectation?
+    func showError() {
+        expectation?.fulfill()
+    }
```

Comment:
Probably a `showErrorCalled: Bool` flag will do here?
Or is there any async operation involved here?

Reply:
Yeah true, I just saw this line a few tests above it and I wanted to use it as it looked cool😂